zerver/actions/realm_settings.py

```diff
@@ -104,6 +105,43 @@ def do_set_realm_property(
         update_users_in_full_members_system_group(realm, acting_user=acting_user)
 
 
+@transaction.atomic(savepoint=False)
```

Comment:
This should have `durable=True` and not `savepoint=False` since we do not call this function inside another atomic transaction as of now.